learningresources/migrations/0001_initial.py

```diff
@@ -20,7 +20,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
```

Comment:
In general, you're correct. At this point, it's fine to delete the dummy DB on the Heroku server, as per @carsongee. During development it's best to try to collapse migrations as much as possible because they quickly get out of hand.